core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.ts

```diff
@@ -47,9 +63,17 @@ export class DotExperimentsConfigurationVariantsComponent {
 
     @Input() stepStatus: StepStatus;
     @Input() variants: Variant[];
-
     @Output() sidebarStatusChanged = new EventEmitter<SidebarStatus>();
     @Output() delete = new EventEmitter<Variant>();
+    @Output() edit = new EventEmitter<Pick<DotExperiment, 'name' | 'id'>>();
     @Output() save = new EventEmitter<Pick<DotExperiment, 'name'>>();
     @Output() goToEditPage = new EventEmitter<{ variant: Variant; mode: EditPageTabs }>();
+    @ViewChildren(Inplace) private inplaceInputs: QueryList<Inplace>;
```

Comment:
I don't see we are using this. Remove it if not needed unless it is a work in progress.